### ISSUE TITLE
Support running via `npx`

### ISSUE THREAD

**Issue by nilskattenbeck-bosch** [novnc/noVNC#1828]
**Is your feature request related to a problem? Please describe.**
I am using a Linux distribution which does not use snapd and would prefer not to install snapd for a single application. Furthermore I also use noVNC on a windows machine from time to time where using snapd is not possible.

**Describe the solution you'd like**
`npx` is a great way to quickly install and run npm projects. To support this it seems like [adding an entrypoint in the bin field of package.json](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin) should be sufficient to do this.

**Describe alternatives you've considered**
Support other standards for software distribution. Flatpak is the most popular variant here and (apart from Ubuntu) supported by all as well as preinstalled on most Linux distributions. This would still be easier than shipping separate Packages for dpkg, rpm, pacman, etc... It would, however, still not resolve the problem of using this on windows.

**Comment by CendioOssman** [comment]
I'm afraid our npm packaging is just the core libraries. It's not a fully functional client, as noVNC needs to be served by a web server.

The snap is a bit odd in that it also packages our sister project, websockify. For clarity, the snap should perhaps be moved to its own repo. But for now, it lives inside the noVNC repo.

I'm also unsure if we really want to add more packagings like that. websockify is a very limited web server, so it's not that appropriate to use for production systems.

In other words, a fully functional service is currently out of scope of noVNC. We're a component you're supposed to combine with other infrastructure. E.g. install Apache and configure it to serve noVNC.

We can certainly explore more ways to make that easier, but more packagings of websockify is not the best way forward, IMO.
